src/lib.rs

```diff
@@ -421,6 +424,17 @@ impl TxBuilder {
         })
     }
 
+    fn set_recipients(&self, recipients: Vec<AddressAmount>) -> Arc<Self> {
+        let recipients = recipients
+            .iter()
+            .map(|address_amount| (address_amount.address.clone(), address_amount.amount))
```

Comment:
Should the `.clone()` method be called on the `amount` as well here?

Reply:
For `address_amount` it is a native u64 and native values implement the rust `Copy` trait which means the value will be implicitly copied and therefore no `clone()` required.